.jenkins/pytorch/build-mobile-code-analysis.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# DO NOT ADD 'set -x' not to reveal CircleCI secret context environment variables
+set -eu -o pipefail
+
+# This script builds and runs code analyzer tool to generate aten op dependency
+# graph for custom mobile build.
+
+COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}"
+
+source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+
+echo "Clang version:"
+clang --version
+
+# !!! THIS HACK SHOULD BE MOVED TO DOCKER BUILD SCRIPT !!!
```

Comment:
Yes please :)

Reply:
Can I land this PR for now before moving it to docker image? I will add it to the docker image for sure but it's blocked by the stacked PR for a while... :>